packages/plugin-heading/Heading.ts

```diff
@@ -1,9 +1,47 @@
 import { JWPlugin } from '../core/src/JWPlugin';
 import { ParsingContext, ParsingMap } from '../core/src/Parser';
 import { HeadingNode } from './HeadingNode';
+import { ParagraphParams } from '../core/src/CorePlugin';
+import { distinct } from '../utils/src/utils';
+import { VNode } from '../core/src/VNodes/VNode';
 
 export class Heading extends JWPlugin {
     readonly parsingFunctions = [this.parse.bind(this)];
+    commandHooks = {
+        applyParagraph: this.applyParagraph.bind(this),
+    };
+    shortcuts = [
+        {
+            pattern: 'CTRL+SHIFT+<Digit1>',
+            commandId: 'applyParagraph',
+            commandArgs: { nodeCreator: (): HeadingNode => new HeadingNode(1) },
+        },
+        {
+            pattern: 'CTRL+SHIFT+<Digit2>',
+            commandId: 'applyParagraph',
+            commandArgs: { nodeCreator: (): HeadingNode => new HeadingNode(2) },
+        },
+        {
+            pattern: 'CTRL+SHIFT+<Digit3>',
+            commandId: 'applyParagraph',
+            commandArgs: { nodeCreator: (): HeadingNode => new HeadingNode(3) },
+        },
+        {
+            pattern: 'CTRL+SHIFT+<Digit4>',
+            commandId: 'applyParagraph',
+            commandArgs: { nodeCreator: (): HeadingNode => new HeadingNode(4) },
+        },
+        {
+            pattern: 'CTRL+SHIFT+<Digit5>',
+            commandId: 'applyParagraph',
+            commandArgs: { nodeCreator: (): HeadingNode => new HeadingNode(5) },
+        },
+        {
+            pattern: 'CTRL+SHIFT+<Digit6>',
+            commandId: 'applyParagraph',
+            commandArgs: { nodeCreator: (): HeadingNode => new HeadingNode(6) },
+        },
+    ];
```

Comment:
Why not `applyHeading` then give `level` as argument ? And in `Paragraph` you would have `applyParagraph` with no argument ? Why do they have to share a common api ? That being said, I understand that it has a certain value of unifying the commands, which is great. Then we might want to discuss about the name `applyParagraph`. Google Docs call that tool "heading style" so I suggest `applyHeadingStyle` ? In my opinion, it makes sense that this command would only be provided by the `Heading` plugin, since it makes no sense to have it if you only have `Paragraph`. Of course that would make `Heading` depend on `Paragraph` but I don't see that as an issue. Tell me if you disagree. Then the `applyHeadingStyle` command could instantiate a `Paragraph` when argument `level` is `0`, so no more `nodeCreator` !